Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -280,43 +290,48 @@ class HTTP1ConnectionProvider {
         self.state.assertInvariants()
     }
 
-    private func execute(_ action: Action) {
+    private func execute(_ action: Action, logger: Logger) {
         switch action {
         case .lease(let connection, let waiter):
             // if connection is became inactive, we create a new one.
             connection.cancelIdleTimeout().whenComplete { _ in
                 if connection.isActiveEstimation {
+                    logger.debug("reusing active connection \(connection)")
                     waiter.promise.succeed(connection)
                 } else {
+                    logger.debug("opening fresh connection (found \(connection) but we assume it seems inactive)")
```

Comment:
either 
```suggestion
                    logger.debug("opening fresh connection (found \(connection) but we assume it inactive)")
```

or

```suggestion
                    logger.debug("opening fresh connection (found \(connection) but it seems inactive)")
```

"we asume it seems"  reads a bit weird; either we "assume it is" or it "seems"

Reply:
Ditto.

Reply:
thanks! done

Reply:
Jumping on this line for different feedback: I asked about this in a merge request from [Cory in RediStack](https://gitlab.com/Mordil/RediStack/-/merge_requests/116/diffs#note_352416226): when to attach dynamic values as metadata vs. embedding in the message itself

It might not be as important to be 100% consistent across the ecosystem, but it's good food for thought as a community to consider

Reply:
🤦 Just saw that this was already discussed below in a different review